dataactvalidator/config/supervisord.conf

```diff
@@ -1,17 +1,18 @@
-[program:uwsgi]
-command=uwsgi --ini /data-act/backend/dataactvalidator/config/uwsgi.ini
+[program:data_act_validator_app]
+process_name=%(program_name)s_%(process_num)02d
 autostart=true
 autorestart=true
 stopasgroup=true
 killasgroup=true
+numprocs=13
 directory=/data-act/backend/dataactvalidator/
 startsecs=1
 startretries=10
-user=ec2-user
+user=root
```

Comment:
This doesn't seem right -- is there a way to run as ec2-user instead? I think that's how all of the ansible stuff is working.